packages/stream_chat_persistence/lib/src/db/shared/native_db.dart

```diff
@@ -0,0 +1,89 @@
+//ignore_for_file: public_member_api_docs
+
+import 'dart:io';
+import 'dart:isolate';
+import 'package:moor/ffi.dart';
+import 'package:moor/isolate.dart';
+import 'package:moor/moor.dart';
+import 'package:path/path.dart';
+import 'package:path_provider/path_provider.dart';
+
+import '../moor_chat_database.dart';
+
+class SharedDB {
```

Comment:
Please add doc comments to this class. Omitting doc comments for public apis will result in a low pub.dev score.